package.json

```diff
@@ -117,10 +117,12 @@
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
     "expensify-common": "2.0.109",
-    "expo": "51.0.31",
-    "expo-av": "14.0.7",
-    "expo-image": "1.12.15",
-    "expo-image-manipulator": "12.0.5",
+    "expo": "52.0.14",
+    "expo-asset": "^11.0.1",
```

Comment:
What do we need this one for?

Reply:
Expo depends on it. IIRC when bumping expo for RN 0.76, when I was building app without explicitly adding it here, it did not build for some reason. I can see that expo has dependency on it: https://github.com/expo/expo/blob/1a16bb66dcd8b0cc7391d9611d923fd54803af3c/packages/expo/package.json#L81 so I am not sure why it didn't work then. 

We can try to remove it in follow-up and maybe it will be properly resolved from expo. It won't change much as for the contents of the app since it is required nevertheless.

Reply:
Yeah lets keep an eye on this so we dont increase the bundle size unnecessary